server/settings/publish_plugins.py

```diff
@@ -350,39 +350,72 @@ class PublishPluginsModel(BaseSettingsModel):
 
 DEFAULT_DEADLINE_PLUGINS_SETTINGS = {
     "CollectJobInfo": {
-        "profiles": [
-            {
-              "group": "",
-              "priority": 50,
-              "job_delay": "",
-              "overrides": [
-                "department",
-                "chunk_size",
-                "group",
-                "priority",
-                "primary_pool",
-                "secondary_pool"
-              ],
-              "chunk_size": 999,
-              "department": "",
-              "host_names": [],
-              "task_names": [],
-              "task_types": [],
-              "limit_groups": [],
-              "machine_list": [],
-              "multiprocess": False,
-              "primary_pool": "",
-              "machine_limit": 0,
-              "use_published": True,
-              "secondary_pool": "",
-              "concurrent_tasks": 1,
-              "machine_list_deny": False,
-              "additional_job_info": "",
-              "additional_plugin_info": "",
-              "use_asset_dependencies": True,
-              "use_workfile_dependency": True
-            }
-        ]
+      "profiles": [
+        {
+          "group": "",
+          "priority": 50,
+          "job_delay": "",
+          "overrides": [
+            "department",
+            "chunk_size",
+            "group",
+            "priority",
+            "primary_pool",
+            "secondary_pool"
+          ],
+          "chunk_size": 1,
+          "department": "",
+          "host_names": [
+            "maya",
+            "houdini",
+            "3dsmax",
+            "blender"
+          ],
+          "task_names": [],
+          "task_types": [],
+          "limit_groups": [],
+          "machine_list": [],
+          "multiprocess": False,
+          "primary_pool": "",
+          "machine_limit": 0,
+          "use_published": True,
+          "secondary_pool": "",
+          "concurrent_tasks": 1,
+          "machine_list_deny": False,
+          "additional_job_info": "",
+          "additional_plugin_info": "",
+          "use_asset_dependencies": False,
+          "use_workfile_dependency": True
+        },
+        {
+          "group": "",
+          "priority": 50,
+          "job_delay": "",
+          "overrides": [],
+          "chunk_size": 9,
```

Comment:
Agree with @m-u-r-p-h-y - this should be 10, not 9. 9 sounds like an odd default.
```suggestion
          "chunk_size": 10,
```

Reply:
Ok, makes sense, updated.